Assets/Scripts/UI/MainMenuScript.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace UI
+{
+    public class MainMenuScript : MonoBehaviour
+    {
+        public void PlayGame()
+        {
+            SceneManager.LoadScene(1);
+        }
+
+        public void QuitGame()
+        {
+            Debug.Log("The game has been quit");
+            Application.Quit();
```

Comment:
Application.Quit doesn't work in the editor, to fix this use the following method :

```c#
#if UNITY_EDITOR
         // Application.Quit() does not work in the editor so
         // UnityEditor.EditorApplication.isPlaying need to be set to false to end the game
         UnityEditor.EditorApplication.isPlaying = false;
#else
         Application.Quit();
#endif
```

Reply:
Addressed